content/blog/2022-07-26-countr-upgrade-maintenance-announcement.mdx

```diff
@@ -0,0 +1,49 @@
+* **Stability improvements:** The uptime and stability is improved as it's now using the latest Discord.js version. It is now also a lot more scalable and easier to upgrade in the future.
+    * *Technical explanation: our approach before was to spawn a worker thread per shard, but we used this way before it scaled to 120 shards. We will now spawn 12 clusters (same as 12 workers) with 10 shards each, and we can easily scale this in the future. It's also now dockerized, so if one cluster crashes then it will auto-restart, versus using pm2 that just restarts all 120 worker shards. Aka. a very nice improvement we should've done a long time ago.*
+* **Multiple counting channels + threads:** You can now make multiple counting channels for your server! You can also now use threads for your counting channels!
+* **Support for other number systems:** Up until now you've only been able to count with [the decimal number system][1], the most standard counting system for us. Well, we introduce [hexadecimal][2], [binary][3], [base36][4], [base64][5], [roman][6] and even counting in math equations!
+* **Flow creator/editor improvements:** The flow editor previously created a channel where you can create a flow. We've rewritten it using slash commands and interactions instead. [Take a sneak peek!](/docs/counting-systems)
+* **Export and import flows:** You can also now export your flow configuration and import it in another server or share it with others!
+
+[1]: https://en.wikipedia.org/wiki/Decimal
+[2]: https://en.wikipedia.org/wiki/Hexadecimal
+[3]: https://en.wikipedia.org/wiki/Binary
+[4]: https://en.wikipedia.org/wiki/Base_36
+[5]: https://en.wikipedia.org/wiki/Base_64
+[6]: https://en.wikipedia.org/wiki/Roman_numerals
+
+## Countr rewritten for the third time
+
+I remember back in the days when Countr used Discord.js version 11 and I was starting to make a rewrite to support version 12. The week after we launched our new version, Discord.js launched version 13. That was quite frustrating to see.
+
+So, we kept going with version 12 until it eventually was going to be unsupported soon. I started rewriting it in version 13, this time in TypeScript so it's much easier to maintain and also upgrade to future upgrades.
+
+The code was almost done, but then - even more frustratingly than the last time - Discord.js launched a development version of version 14.
+
+I decided to scrap the code and rewrite it again. I also had better ideas for the code anyways so I would either have to rewrite every file in the project, or just start from scratch again. I am quite happy with how this code turned out though, and Discord.js also released version 14 which we upgraded to.
+
+## Website overhaul
+
+We've changed up our website and made it easier to maintain and also moved documentation over instead of having two separate websites.
+
+**You can also now contribute to the website by adding tutorials or community posts!**
```

Comment:
Add a link of where to contribute eg [contribute](https://github.com/countr/website/#contribute=)